spring/boot2-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/WebOperationServiceUtil.java

```diff
@@ -0,0 +1,16 @@
+package com.linecorp.armeria.spring.actuate;
```

Comment:
missing copyright?

Reply:
😱 There was a problem with my IntelliJ IDEA so I clean up the configuration. 😅